scripts/sync_erddap_datasets.py

```diff
@@ -1,4 +1,7 @@
 #!/usr/bin/env python
+#
+# THIS SCRIPT IS DEPRECATED. SYNCING ERDDAP IS DONE IN glider_dac_watchdog.py
```

Comment:
once we're happy with this update I will remove this script from source control.